docs/products/mysql/howto/connect-with-python.rst

```diff
@@ -0,0 +1,54 @@
+Connect to MySQL with Python
+============================
+
+This example connects your Python application to a MySQL service, using the `PyMySQL <https://github.com/PyMySQL/PyMySQL>`__ library.
+
+Variables
+'''''''''
+
+These are the placeholders you will need to replace in the code sample:
+
+==================      =============================================================
+Variable                Description
+==================      =============================================================
+``MYSQL_HOST``          Host name for the connection, from the service overview page
+------------------      -------------------------------------------------------------
+``MYSQL_PORT``          Port number to use, from the service overview page
+------------------      -------------------------------------------------------------
+``MYSQL_USERNAME``      User to connect with
+------------------      -------------------------------------------------------------
+``MYSQL_PASSWORD``      Password for this user
+==================      =============================================================
+
+Pre-requisites
+''''''''''''''
+
+For this example you will need:
+
+* Python 3.6 or later
```

Comment:
off-topic: Python 3.6 will reach end of life next week (23 Dec).

Reply:
bumped to 3.7